notebooks/visualize/visualize.ipynb

```diff
@@ -0,0 +1,1034 @@
+{
```

Comment:
<p>This isn't needed once you've created the file in the first ever notebook. You should delete this.</p><p>If before you start the jupyter lab or jupyter notebook you have the env var "MSTICPYCONFIG" set to point to the msticpyconfig.yaml it will automatically be loaded.</p><p>If the env var isn't set it will look in the current directory for msticpyconfig.yaml</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/mstic-dataexpl/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>